easybuild/tools/github.py

```diff
@@ -703,24 +710,25 @@ def fetch_files_from_commit(commit, files=None, path=None, github_account=None,
 
     # if no files are specified, determine which files are touched in commit
     if not files:
-        diff_url = os.path.join(GITHUB_URL, github_account, github_repo, 'commit', commit + '.diff')
-        diff_fn = os.path.basename(diff_url)
-        diff_filepath = os.path.join(path, diff_fn)
-        # max. 6 attempts + initial wait time of 10sec -> max. 10 * (2^6) = 640sec (~10min) before giving up on download
-        # see also https://github.com/easybuilders/easybuild-framework/issues/4869
-        max_attempts = DEFAULT_DOWNLOAD_MAX_ATTEMPTS
-        download_file(diff_fn, diff_url, diff_filepath, forced=True, trace=False,
-                      max_attempts=max_attempts)
-        if os.path.exists(diff_filepath):
-            diff_txt = read_file(diff_filepath)
+        github_user = build_option('github_user')
+
+        def pr_request_fn(gh):
+            return gh.repos[github_account][github_repo].pulls[pr]
```

Comment:
this should be `commit_request_fn` and return the commit diff

Reply:
last minute refactoring gone wrong... 🤦‍♂️
fixed in fce140bcc